src/bin/nydus-image/inspect.rs

```diff
@@ -51,9 +54,34 @@ impl RafsInspector {
             bootstrap: Arc::new(Mutex::new(f)),
             cur_dir_ino: root_ino,
             parent_inoes: Vec::new(),
+            files_parent: BTreeMap::new(),
```

Comment:
Can you please fix the typo `parent_inoes` too ?

Reply:
Yes, fixed it.